### PR TITLE
docs: create railroad diagrams for SPLIT AT

### PR DESCRIPTION
**Pull request by nvb**
See https://github.com/cockroachdb/docs/issues/616.

Release note: None

### CODE REVIEW

**Review by benesch — APPROVED** [review]
Can you run `make build` and commit the generated diffs in `docs/generated/sql/bnf/`?

**Comment by nvanbenschoten** [comment]
> Can you run make build and commit the generated diffs in docs/generated/sql/bnf/?

Done.

File: pkg/cmd/docgen/diagrams.go [hunk 1]
```diff
@@ -822,6 +822,8 @@ var specs = []stmtSpec{
 	{name: "show_trace", stmt: "show_trace_stmt"},
 	{name: "show_transaction", stmt: "show_stmt", match: []*regexp.Regexp{regexp.MustCompile("'SHOW' 'TRANSACTION'")}},
 	{name: "show_users", stmt: "show_stmt", match: []*regexp.Regexp{regexp.MustCompile("'SHOW' 'USERS'")}},
+	{name: "split_index_at", stmt: "alter_split_index_stmt"},
```
Comment (benesch) [inline pkg/cmd/docgen/diagrams.go]: For consistency with `ALTER INDEX ... RENAME` you might want to apply the following config:

```
inline: []string{"table_name_with_index"}
replace: map[string]string{"qualified_name": "table_name", "'@' name": "'@' index_name"},
```

Comment (nvanbenschoten) [reply pkg/cmd/docgen/diagrams.go]: Good point! Done.

**Review by benesch — APPROVED** [review]
Sorry about the CI failure! It's upset because I'd accidentally deleted all the docs when this ran. 🙈

**Pull request merged**